Makefile

```diff
@@ -0,0 +1,25 @@
+BATS := "node_modules/bats/bin/bats"
+
+setup: install-deps launch_registry
+
+install-deps:
+	npm install bats@1.x
```

Comment:
installation via NPM is weird for a bash tool, but it's suggested by their README, so who am I to judge

Reply:
While we use JS elsewhere, I have a preference for using the Docker image.

Reply:
I agree. For me a single `npm intall` feels odd but cleaner than dealing with nested docker or curl && ./install.sh

Reply:
There was a constraint on not doing nested docker, because the mounting of folders and ports would make it pretty complicated.